### ISSUE TITLE
Unlock 180° screen rotation support

### ISSUE THREAD

**Issue by omitter** [wireapp/wire-android#1570]
I use my phone quite a lot in 180° rotated portrait mode (easier charging). Many apps support this, but enough do not. Wire is one of those. Whenever I open Wire, it rotates the screen back to 0° portrait mode, which is really frustrating as I have to rotate the phone to use the app. I suppose implementing this is much easier than implementing a landscape mode. Please consider supporting this feature.

Wire-Version: 3.10.439
Android: 7.1 (LineageOS 14.1)

**Comment by jpotier** [comment]
I actually would make a lot of sense to allow typing in landscape mode, as it is easier on some phones.

**Comment by svaens** [comment]
Yes, please. Support this. I can not use Wire without this.

**Comment by madranet** [comment]
I've given you a "plus one" but don't hold your breath. I've been asking for this for years and got no positive response from the devs at all. 

Being forced to use portrait layout when you've locked your device to Landscape is jarring enough [and terrible user experience!] but the fact that even the portrait layout is so badly designed for text editing just makes it even more annoying. 

Out of the entire height of the screen, you literally get one line for text input and editing on my phone and 2 lines on my tablet. I really wish someone at Wire would take this usability issue seriously but, on the rare occasions when I have managed to elicit a response, all I've got is a bald statement of fact; _"Wire doesn't support tablets"_ or _"Wire doesn't support landscape mode"_ with no sense that they're even considering this.
